src/KubernetesClient.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <PackageVersion>0.2.0-beta</PackageVersion>
+    <PackageVersion>0.2.1-beta</PackageVersion>
```

Comment:
maybe should be managed by @brendandburns

Reply:
We've created a NuGet package with those changes and I have let it there by mistake. 
Fixed